seshdash/views.py

```diff
@@ -240,3 +254,39 @@ class UserList(generics.ListAPIView):
 class UserDetail(generics.RetrieveAPIView):
     queryset = User.objects.all()
     serializer_class = UserSerializer
+
+
+def get_high_chart_data():
+    # this the content object
+     content_data = {}
+    # Initializing Date
+     now = datetime.datetime.now()
+     five_day_past2 = now - timedelta(days=5)
+     five_day_future2 = now + timedelta(days=6)
+    # Queries and Data retreiving of the climate
+     high_cloud_cover = Site_Weather_Data.objects.filter(date__range=[five_day_past2,five_day_future2]).values_list('cloud_cover', flat=True).order_by('date')
+     content_data['high_cloud_cover']=high_cloud_cover
+    # Queries and Data retreiving of the PV Production and Dates
+
+     high_date = Site_Weather_Data.objects.filter(date__range=[five_day_past2,five_day_future2]).values_list('date', flat=True).order_by('date')
+     high_date2 = []
+     last_date=None
+     high_pv_production =[]
+     for elt in high_date:
+        date1=elt.strftime("%d %B %Y")
+        high_date2.append(date1)
+        if last_date==None:
+            last_date= elt
+
+        pv_sum_day= BoM_Data_Point.objects.filter(time__range=[last_date,elt]).aggregate(pv_sum=Sum('pv_production'))
+        last_date=elt
+
+        pv_sum_day2 = pv_sum_day.get('pv_sum', 0)
```

Comment:
what is pv_sum_day2? Why 2? 

It's important to name variables with meaningful name. What's the difference between pv_sum_day and pv_sum_day2. It's not obvious and there isn't a comment explaining there should be a comment every time you'r doing something meaningful l within the code.